wagtail/documents/static_src/wagtaildocs/js/document-chooser-modal.js

```diff
@@ -11,6 +11,18 @@ DOCUMENT_CHOOSER_MODAL_ONLOAD_HANDLERS = {
                 setPage(page);
                 return false;
             });
+
+            $('a.upload-one-now').on('click', function(e) {
+                // Set current collection ID at upload form tab
+                let collection_id = $('#collection_chooser_collection_id').val();
```

Comment:
looks like in wagtain js code you need to user camelCase for var names
@thibaudcolas @jjanssen  what do you think ?

Reply:
Honestly it wouldn't surprise me if there would be a mix & match of all sorts of notations within our current javascript definitions. As our initial javascript linting accidentally got disabled some time ago (#2797). 

Looking at our more recent work regarding the integration of the Explorer menu and Draftail in React we enforce camelCasing in our variable names. But since this is a bit of legacy I'm ok with leaving this as is.